### ISSUE TITLE
Alt text is not descriptive [CLI pages]

### ISSUE THREAD

**Issue by timngyn** [aws-amplify/docs#7050]
**Describe the content issue**:
Images listed below do not have descriptive/informative alt text. For markdown the alt text is located between the square brackets, `![alt text](...)`.

The list below isn't exhaustive since I found these images based on the word count for the alt text, but it should be a good starting point to make our alt text better for people who use screen readers.

Some resources to help write alt text:
- https://webaim.org/techniques/alttext/
- https://www.a11yproject.com/posts/are-you-making-these-five-mistakes-when-writing-alt-text/

https://github.com/aws-amplify/docs/blob/main/src/pages/[platform]/tools/cli/index.mdx:
```
  53  
  54: ![Amplify CLI feature carousel](/assets/cli-b-roll.gif)
  55  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/pages/[platform]/tools/cli/hosting/index.mdx:
```
  95  
  96: ![SPA redirect](/images/hosting/spa-redirect.png)
  97  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/pages/[platform]/tools/cli/migration/aws-cdk-migration/index.mdx:
```
  83  
  84:    ![Amplify overrides warning](/images/ts-guide-cdk-package.png)
  85     And update amplify/backend/custom/<custom-resource-name>/cdk-stack.ts for custom categories as shown below:
  86      
  87:    ![Amplify custom warning](/images/ts-guide-cdk.png)
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/pages/[platform]/tools/cli/plugins/architecture/index.mdx:
```
  54  
  55: ![Image](/images/plugin-platform.png)
  56  

  77  
  78: ![Image](/images/AmplifyCliConcept.jpg)
  79  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/pages/[platform]/tools/cli/project/troubleshooting/index.mdx:
```
  228  
  229: ![Amplify GraphQL schema validation error](/images/ts-guide-gql-schema-validation-error.png)
  230  

  294  
  295: ![Amplify GraphQL model deletion warning](/images/ts-guide-model-deletion-warning.png)
  296  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/pages/[platform]/tools/cli/teams/index.mdx:
```
  52  
  53: ![Image](/images/AmplifyEnvSwitching.jpg)
  54  

  56  
  57: ![Image](/images/AmplifyEnvAddDeploy.jpg)
  58  

  60  
  61: ![Image](/images/AmplifyEnvAddDeploySwitching.jpg)
  62  

  64  
  65: ![Image](/images/AmplifyEnvMultDevelopers.jpg)
  66  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/pages/[platform]/tools/cli/teams/multi-frontend/index.mdx:
```
  54  
  55: ![Image](/images/multiple-frontends.png)
  56  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/pages/[platform]/tools/cli/usage/containers/index.mdx:
```
  453  
  454: ![Fully Managed Pipeline](/images/containers/build-workflow.png)
  455  
```
